test/rules/access-on-weekdays-only-for-an-app.test.js

```diff
@@ -0,0 +1,44 @@
+const loadRule = require('../utils/load-rule');
+
+describe('access-on-weekdays-only-for-an-app', () => {
+  let globals;
+  let user;
+  let context;
+  let rule;
+
+  beforeEach(() => {
+    globals = {
+      UnauthorizedError: function() {}
+    };
+    context = {
+      clientName: 'TheAppToCheckAccessTo'
+    };
+
+    rule = loadRule('access-on-weekdays-only-for-an-app.js', globals);
```

Comment:
let's just set a `const` at the top of test with the `testName`. Then you can use that as the `describe` message.